mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.FLV.Mixfile do
   use Mix.Project
 
-  @version "0.4.0"
+  @version "0.4.1"
```

Comment:
Hmm I think we are breaking the API since now we will be using different h264 format - shouldn't we bump the minor?

Reply:
The change in H264 format, according to the diff, is only making one of the fields optional. Plugins upgrading from the previous version should be setting said field, so unless it changes in the update, this is not a breaking change of their API.

Reply:
Well, then probably we shouldn't have bumped the minor in h264_format then :D